src/transport/raw/UDP.h

```diff
@@ -71,11 +75,20 @@ class UdpListenParameters
         return *this;
     }
 
+    void * GetNativeParams() const { return mNativeParams; }
```

Comment:
Some comments here and wherever other "Native Params" are used would be helpful. A reader will for sure question "what are native parameters" when looking through the code. I am not sure if we can express more via naming, like  "NativeNetworkStackParams".

Reply:
Done